### ISSUE TITLE
callback firing before expected.

### ISSUE THREAD

**Issue by MarcoMedrano** [moarwick/react-mt-svg-lines#15]
Hi it is a very cool lib!! Just trying current version. the issue is callback function is fired almost immediately rather than at the end of the animation.

**Comment by moarwick** [comment]
hi @MarcoMedrano. sorry, just noticed the issue. can you tell me your props and which browser? the callback should be fired after `start delay + (anim duration * iterations)`, but in IE it falls back to Tween.js which has its own `onComplete`...

**Comment by rmaksim** [comment]
Hello.
I have same issue.

duration={xxx} 
playback is not defined

```
static defaultProps = {
    ...
    playback: 'forwards',
}
...
let numOfRepeats = parseInt(playback, 10) || 0
...
setTimeout(..., startDelay + duration * (numOfRepeats) )
```

0 + xxx * 0 = 0

**Comment by moarwick** [comment]
hmm, that indeed looks wrong! checking...

**Comment by moarwick** [comment]
duh, definitely had an issue there but should be fixed in 0.8.4, see [Demo](http://moarwick.github.io/react-mt-svg-lines/)
